.github/workflows/at_server.yaml

```diff
@@ -458,11 +456,11 @@ jobs:
         working-directory: ${{ env.e2etest-working-directory }}
         run: dart pub get
 
-      - name: Cloning at_libraries
+      - name: Cloning at_client_sdk
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          repository: atsign-foundation/at_libraries
-          path: at_libraries
+          repository: atsign-foundation/at_client_sdk
+          path: at_client_sdk
```

Comment:
The only functional change to this file, I can revert the formatting if needed. I wasn't paying attention and formatted the file.